charts/redpanda/templates/statefulset.yaml

```diff
@@ -275,7 +275,7 @@ spec:
             - start
             - --smp={{ .Values.resources.cpu.cores }}
             - --memory={{ template "redpanda-memory" . }}M
-            - --reserve-memory={{ template "redpanda-reserve-memory" . }}
+            - --reserve-memory={{ template "redpanda-reserve-memory" . }}M
```

Comment:
The values file describe this unit, and should it always be an M?

Reply:
Yes, we convert it in the template from SI units to a int64 (in MiB)